tools/src/commands/GenerateBareApp.ts

```diff
@@ -291,6 +302,33 @@ async function createScripts({ projectDir }) {
   console.log('Added package scripts!');
 }
 
+async function stageAndCommitInitialChanges({ projectDir }) {
+  await spawnAsync('git', ['init'], { cwd: projectDir });
+  await spawnAsync('git', ['add', '.'], { cwd: projectDir });
+  await spawnAsync('git', ['commit', '-m', 'Initialized bare app!'], { cwd: projectDir });
+}
+
+async function modifyAppJson({ projectDir, appName }: { projectDir: string; appName: string }) {
+  const pathToAppJson = path.resolve(projectDir, 'app.json');
+  const json = await fs.readJson(pathToAppJson);
+
+  const strippedAppName = removeCharRecursive(appName, '-');
+  json.expo.android = { package: `com.${strippedAppName}` };
+  json.expo.ios = { bundleIdentifier: `com.${strippedAppName}` };
+
+  await fs.writeJSON(pathToAppJson, json, { spaces: 2 });
+}
+
+function removeCharRecursive(str: string, charToReplace: string) {
+  let copy = str;
+
+  while (copy.includes(charToReplace)) {
+    copy = copy.replace(charToReplace, '');
+  }
```

Comment:
Just `str.replaceAll(charToReplace, '')` should do the same thing 🤔

Reply:
is there a specific node version that these tools are tied to? `replaceAll` isn't included in the `@types` package we are using, should I update this?

Reply:
We don't explicitly specify any requirement for the node version, but I think we should set this to at least v16. Looks like `replaceAll` was added in v15.

Reply:
just updated this now - i also needed to update the lib target in tsconfig to get this to work